README.md

```diff
@@ -76,6 +76,12 @@ If all goes well, you will also see a new directory (by default `<your config di
 > (Open file) Ctrl-A (select all) Ctrl-C (copy)
 
 Open Home Assistant and open the LoveLace editor by clicking the elipses on the top right of the screen, and then select "Configure UI". Click the elipses again and click "Raw config editor". Scroll down to the bottom of the screen and then paste your clipboard. This will paste a new View for your lock. Click the Save button then click the X to exit.
+
+In order for the lovelace ui to work properly you will need to install the following modules.
```

Comment:
I'd add a quip about being able to install these via HACS as they are in the default HACS repo for installing.